src/main/java/model/Database/Queries.java

```diff
@@ -7,21 +7,91 @@
 
     public static class Create {
 
-        public static String CREATE_TABLE_RESERVATIONS = ""; // TODO: Replace empty string with query
-
-        public static String CREATE_TABLE_RENTALS = ""; // TODO: Replace empty string with query
-
-        public static String CREATE_TABLE_VEHICLES = ""; // TODO: Replace empty string with query
-
-        public static String CREATE_TABLE_VEHICLE_TYPES = ""; // TODO: Replace empty string with query
-
-        public static String CREATE_TABLE_CUSTOMERS = ""; // TODO: Replace empty string with query
-
-        public static String CREATE_TABLE_RETURNS = ""; // TODO: Replace empty string with query
-
-        public static String CREATE_TABLE_CREDIT_CARDS = ""; // TODO: Replace empty string with query
-
-        public static String CHECK_TABLE_EXISTS = ""; // TODO: Replace empty string with query
+        public static String CREATE_TABLE_RESERVATIONS = "CREATE TABLE IF NOT EXISTS Reservations" +
```

Comment:
I think you're missing open parenthesis in all of these statements

Reply:
Maybe before we merge, copy these statements into the MySql client and see if it throws any errors? Otherwise, they look good to me. Thanks :)

Reply:
All of them work. I tested them on the client

Reply:
It's giving me an error 
![image](https://user-images.githubusercontent.com/35513492/68743260-e7ce7a00-05a6-11ea-90f5-fc3f798593c4.png)

Reply:
You're missing open parenthesis `"("` after the table name in all of them